ansible/roles/spatial-portal/tasks/main.yml

```diff
@@ -0,0 +1,27 @@
+- include: ../../common/tasks/setfacts.yml
```

Comment:
If you tag every task in this file with a 'spatial-portal' tag we can then run the playbook with that tag: this makes the playbook considerably faster if all the software dependencies (java, tomcat, db, etc etc) already exist by just running these tasks.